sdk/core/System.ClientModel/src/Internal/ContentTypeUtilities.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Text;
+
+namespace System.ClientModel.Internal;
+
+internal class ContentTypeUtilities
+{
+    public static bool TryGetTextEncoding(string contentType, out Encoding? encoding)
+    {
+        const string charsetMarker = "; charset=";
```

Comment:
Are we sure that there is always a space between the `;` and `charset=` and that there is no space before the `=` ?

I know that's the general form, but I believe HTTP allows for arbitrary spacing there.

Reply:
This is just copy & pasted from here: https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/src/Shared/ContentTypeUtilities.cs but this is a good callout. I'll look at the implementation more closely and fix in both places if necessary.